test/Microsoft.Azure.Devices.Edge.Test.Common/config/IModuleConfigBuilder.cs

```diff
@@ -11,10 +11,12 @@ public interface IModuleConfigBuilder
 
         IModuleConfigBuilder WithDesiredProperties(IDictionary<string, object> properties);
 
-        IModuleConfigBuilder WithEnvironment(IEnumerable<(string, string)> env);
+        IModuleConfigBuilder WithEnvironment(params (string, string)[] env);
```

Comment:
There are several callers of this that need to be updated, right?

Reply:
nope, passing in an array is valid for a params function.